src/other/demo.canvas/protocol.json

```diff
@@ -233,6 +469,51 @@
     }
   },
   "stages": [
+    {
+      "id": "namegenroster1",
+      "type": "NameGeneratorAutoComplete",
+      "label": "NG Classmates",
+      "form": "autocompletingSearchForm",
```

Comment:
I think we want to rethink the way that this interface handles its relationship to the form in its config. 

In the standard name generator, we want form to be passed because it could contain an arbitrary redux form, that our Architect app will let users create. In the case of the autocomplete, there doesn't seem to be any customisation possible with this form.

Can we hardwire this in, for now, and remove the config property?

Reply:
Oops, this is a vestige of my original prototype. I removed the protocol-driven form based on conversations with @wwqrd ; just need to remove this reference.